src/amo/components/RatingManager/index.js

```diff
@@ -18,34 +21,54 @@ import {
 import translate from 'core/i18n/translate';
 import log from 'core/logger';
 import DefaultRating from 'ui/components/Rating';
+/* eslint-disable no-duplicate-imports */
```

Comment:
This plugin can distinguish between flow imports and normal ones: see https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/no-duplicates.md